packages/web/components/ui/media-audio-node-static.tsx

```
@@ -1,15 +1,17 @@
+} from 'platejs'
 
 export function AudioElementStatic(props: SlateElementProps<TAudioElement>) {
   return (
-    <SlateElement {...props} className="mb-1">
-      <figure className="group relative cursor-default">
-        <div className="h-16">
-          <audio className="size-full" src={props.element.url} controls />
+    <SlateElement {...props} className='mb-1'>
+      <figure className='group relative cursor-default'>
+        <div className='h-16'>
+          <audio className='size-full' src={props.element.url} controls />
```

Biome (1.9.4)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)